### PR TITLE
Update help.txt

### PR DESCRIPTION
**Pull request by timrae**
Make the bug report procedure more clear, and improve the page formatting

### CODE REVIEW

**Comment by timrae** [comment]
@snowtimeglass 
I made some changes to the English help page

**Comment by snowtimeglass** [comment]
Thank you.
I'll translate the changes before long.

File: help.txt [hunk 1]
```diff
@@ -5,28 +5,28 @@
...
 4. Tap the "copy debug info" button at the bottom (which copies "debug info" to the clipboard)
-5. Paste the contents of the clipboard to the issue
+5. If filling out the bug report on your mobile device, paste the contents of the clipboard directly to the issue. Alternatively, you can paste the contents of the clipboard into a new email, send it to yourself, then copy and paste that into the bug report from your computer.
 
 ++++++++++
 <img src="img/DebugInfo.png">
 ++++++++++
 
-== Contributing to AnkiDroid
+This "debug info" is important as it allows us to match your report with our internal crash report data.
```
Comment (snowtimeglass) [inline help.txt]: Does "our internal crash report data" mean the following data?
http://ankidroid-triage.appspot.com/report_crashes 

Does this description fit even a situation in which an user sets "Error reporting modes" to "Never report"?

Comment (timrae) [inline help.txt]: No the crash data is now on our private server. Of course if they set it to
never then we won't have the data, but it's not worth explaining in that
much detail

**Comment by snowtimeglass** [comment]
I understand. 
Thank you!

**Pull request merged**